src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  static alive = [];
+  static deleteIndex = Animal.alive.findIndex((animal) => animal.health <= 0);
+  constructor(health, name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  deleteAnimale() {
```

Comment:
The method name `deleteAnimale` has a typo and does not follow the function naming convention. It should be renamed to `deleteAnimal` to correctly reflect its functionality and improve readability.